### ISSUE TITLE
Heapster standalone documentation

### ISSUE THREAD

**Issue by shilpapadgaonkar** [kubernetes/heapster#299]
When will the standalone heapster documentation be available?

**Comment by rjnagal** [comment]
Can you clarify what you mean by standalone? Without kubernetes or fleet?

**Comment by shilpapadgaonkar** [comment]
Yes. There is a link for the same in the docu https://github.com/GoogleCloudPlatform/heapster/blob/master/docs/standalone.md

**Comment by rjnagal** [comment]
ah, got it. Will try to get to it this week.

**Comment by shilpapadgaonkar** [comment]
Thanks!

**Comment by DreadPirateShawn** [comment]
Any update on this doc? Still empty. Even just a sample command or flags...

**Comment by roshan-zot** [comment]
Is there a half-written documentation somewhere?

**Comment by vishh** [comment]
What are the use cases for standalone mode? Is it for dev testing or having
a scaled out setup?

**Comment by DreadPirateShawn** [comment]
I can't speak for the others, but I arrived at this doc stub while attempting to run heapster as a separate process with `--source` pointing to kubernetes master.

I ended up filing issue #392 during my trial-and-error phase, and ultimately @jimmidyson suggesting adding `inClusterConfig=false` as an extra option to the kube source.

This was indeed the solution to my troubles, and that's the sort of info which I would expect to be in the "standalone" doc.

Granted, there may be other valid contexts for "standalone" heapster -- @rjnagal above questions whether this may apply to running without kubernetes or fleet, and I am definitely running *with* kubernetes. Perhaps the doc should include use-cases for various "standalone" interpretations? eg, "standalone" meaning "running the heapster binary externally to a kubernetes cluster rather than as a container in the cluster", "running the heapster binary externally with a source other than kubernetes" (if that's even a thing that can be done), etc... dev testing vs having a scaled out setup... basically each variation of "standalone" could be acknowledged in the doc, along with either a working example (eg `/my/bin/heapster --v=5 --logtostderr=true --port=8082 --source="kubernes:http://10.0.0.2:8080?insecure=true&inClusterConfig=false"`) or confirmation of lack of support.

Just my two cents, of course.
